values.yaml

```
@@ -563,6 +563,7 @@ st2timersengine:
 # https://docs.stackstorm.com/reference/ha.html#st2workflowengine
 # Multiple st2workflowengine processes can run in active-active mode and will share the load and pick up more work if one or more of the processes become available.
 st2workflowengine:
+  #terminationGracePeriodSeconds: 350
```

```suggestion
  # include some small explanation about this setting
  terminationGracePeriodSeconds: 300
```
Could we set the default here instead and uncomment it? This way it will be easier to manage it, including controlling the defaults.

Please also add a small code comment so users know how it's helpful as it plays together with the st2.conf runner graceful shutdown, right?